File: version1/nodejs/simple_push.js
var request = require('request');

var TOKEN = "YOUR_TOKEN"

var headers = {
    'content-type': 'application/json',
    'Authorization': 'Token ' + TOKEN,
    'Accept':'application/json',
};

var requestData = {
    applications: ["YOUR_APPLICATION_ID"],
    notification: {
        title:"this is the title",
        content: "this is the content",
    }
};

request(
    {
        url: "https://panel.pushe.co/api/v1/notifications/",
        json: true,
        headers: headers,
        method:'POST',
        body: requestData
    },
    function(error, response, body) {
        console.log(response)
        resp.send("err: "+error+" body:"+body)
    }
);

File: version1/cs/simple_push.cs

public partial class Admin_Users : System.Web.UI.Page
{
    protected void Page_Load(object sender, EventArgs e)
    {
        var httpWebRequest = (HttpWebRequest)WebRequest.Create("https://panel.pushe.co/api/v1/notifications/");
        httpWebRequest.ContentType = "application/json";
        httpWebRequest.Accept = "application/json";
        httpWebRequest.Method = "POST";
        httpWebRequest.Headers.Add("Authorization", "Token YOUR_TOKEN"

        string jsonData = "{\"applications\":[\"YOUR_APPLICATION_ID\"],\"notification\":{\"title\": \"title\",\"content\":\"content\"}}";

        using (var streamWriter = new StreamWriter(httpWebRequest.GetRequestStream()))
        {
            streamWriter.Write(jsonData);
            streamWriter.Flush();
            streamWriter.Close();

            var httpResponse = (HttpWebResponse)httpWebRequest.GetResponse();
        }

     }
}
